iamctl/pkg/utils/init.go

```diff
@@ -30,7 +30,10 @@ var dir, _ = os.Getwd()
 var Path = dir + "/iamctl.json"
 var PathSampleSPDetails = dir + "/init.json"
 
-const SCOPE string = "internal_application_mgt_update internal_application_mgt_create internal_application_mgt_view internal_application_mgt_delete internal_idp_update internal_idp_create internal_idp_view internal_idp_delete internal_userstore_view internal_userstore_create internal_userstore_update internal_userstore_delete internal_claim_meta_create internal_claim_meta_view internal_claim_meta_update internal_claim_meta_delete internal_org_idp_delete internal_org_idp_view internal_org_idp_update internal_org_idp_create internal_org_claim_meta_update internal_org_claim_meta_view internal_org_application_mgt_update internal_org_application_mgt_create internal_org_application_mgt_view internal_org_application_mgt_delete internal_org_userstore_create internal_org_userstore_view internal_org_userstore_delete internal_org_userstore_update"
+const SCOPE string = "internal_application_mgt_update internal_application_mgt_create internal_application_mgt_view internal_application_mgt_delete internal_idp_update internal_idp_create internal_idp_view internal_idp_delete internal_userstore_view internal_userstore_create internal_userstore_update internal_userstore_delete internal_claim_meta_create internal_claim_meta_view internal_claim_meta_update internal_claim_meta_delete internal_org_idp_delete internal_org_idp_view internal_org_idp_update internal_org_idp_create internal_org_claim_meta_update internal_org_claim_meta_view internal_org_application_mgt_update internal_org_application_mgt_create internal_org_application_mgt_view internal_org_application_mgt_delete internal_org_userstore_create internal_org_userstore_view internal_org_userstore_delete internal_org_userstore_update " +
+	"internal_application_mgt_client_secret_create  internal_org_application_mgt_client_secret_create internal_application_mgt_client_secret_view internal_org_application_mgt_client_secret_view " +
+	"internal_application_script_update internal_org_application_script_update " +
+	"internal_application_business_api_update internal_application_internal_api_update"
```

Comment:
We need following scopes for org specific operations.

`internal_org_application_business_api_update` and `internal_org_application_internal_api_update`

How ever those scopes are not onboarded to the IS yet. 
If adding a not existing scope will not break the tool, lets add them now. Otherwise we might miss to add them.